spec/core_functions/color/scale_color/error/args.hrx

```diff
@@ -0,0 +1,60 @@
+<===> too_few/input.scss
+a {b: scale-color()}
+
+<===> too_few/error
+Error: Missing argument $color.
+  ,
+1 | a {b: scale-color()}
+  |       ^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===> too_few/error-libsass
+Error: Function scale-color is missing argument $color.
+        on line 1 of input.scss
+>> a {b: scale-color()}
+
+   ------^
+
+<===>
+================================================================================
+<===> too_many/options.yml
+---
+:todo:
+- sass/libsass#2900
+
+<===> too_many/input.scss
+a {b: scale-color(red, 1)}
+
+<===> too_many/error
+Error: Only one positional argument is allowed. All other arguments must be passed by name.
+  ,
+1 | a {b: scale-color(red, 1)}
+  |       ^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> unknown/options.yml
+---
+:todo:
+- sass/libsass#2901
+
+<===> unknown/input.scss
+a {b: scale-color(red, $hue: 10%)}
+
+<===> unknown/error
+Error: No argument named $hue.
+  ,
+1 | a {b: scale-color(red, $hue: 10%)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===> unknown/error-libsass
+Error: Function scale-color has no parameter named $ambience
```

Comment:
This (and below) should be `$hue` to match the input and Dart Sass error

Reply:
We just shouldn't include LibSass errors for specs that are marked as TODO for LibSass.